docs/website/docs/general-usage/credentials/setup.md

```diff
@@ -335,6 +336,8 @@ export DESTINATION__FILESYSTEM__CREDENTIALS__AWS_SECRET_ACCESS_KEY="api_key"
 ```py
 import os
 import dlt
+import botocore.session
+from dlt.common.credentials import AwsCredentials
```

Comment:
This looks nice, but dangerous territory

Reply:
I agree, but the change is good :D